common/views/themes/base/wellcome-normalize.ts

```diff
@@ -22,17 +20,11 @@ fieldset {
   padding: 0;
 }
 
-.openseadragon-canvas:focus {
```

Comment:
just curious; what was this?

Reply:
OpenSeadragon is the library that we use to deal with deep zoom of tiled IIIF Images. We were explicitly giving it a focus state, but since we're now giving everything a focus state, it isn't necessary to call it out individually (although this might change with work in #10299 because this is slightly gnarly with NVDA)

Reply:
…that said, I think I've missed something because there isn't a yellow border there at present – 🤔 thanks! Will fix

Reply:
re-added it (with new variable) because box-shadow gets hidden by any amount of zooming in